### ISSUE TITLE
Convention for specifying dependencies

### ISSUE THREAD

**Issue by jsh9** [Caltech-geoquake/GM-GANO#1]
The convention for specifying dependencies is usually a `requirements.txt` file like this:

```
pytorch
torchvision
torchaudio
pytorch-cuda==11.8
pandas
...
```

And then let people install them by `pip install -r requirements.txt`.

Conda may not be necessary, pip should be sufficient.

Also, it seems torchvision and torchaudio are not needed in the algorithm?  There's no need to specify unused dependencies in the `requirements.txt` file.

**Comment by yzshi5** [comment]
Hi Jian, I agree with that a `requirements.txt` may be necessary and more formal. Since this work is a pilot study, and has several follow ups. I will be more careful in further code release. 

Usually, build a separate conda environment is a convenient way for testing a new model, especially for people without a good background in computer system.  For the installation of unused `torchvision` and `torchaudio`, I just copied the command line for installing pytorch from the official website.

**Comment by jsh9** [comment]
> Usually, build a separate conda environment is a convenient way for testing a new model, especially for people without a good background in computer system.

Yeah, people can just do `pip install -r requirements.txt` after they create a fresh conda environment.
